packages/eas-cli/src/submissions/utils/config.ts

```diff
@@ -0,0 +1,26 @@
+import { ExpoConfig, getConfig } from '@expo/config';
+
+interface AppConfig {
+  owner?: string;
+  slug: string;
+}
+
+export function getAppConfig(projecDir: string): AppConfig {
+  const exp = getExpoConfig(projecDir);
+  if (!exp.slug) {
```

Comment:
I think that slug can't be missing if it's not present in app.json name of the directory is used

Reply:
I didn't dig into `@expo/config` internals, I just assumed this code is correct as it was used in `expo-cli`. In theory, `exp.slug` is of type `string` (non-nullable), but a double check won't hurt.